### DISCUSSION TITLE
New Autograding Tests UI Broke Reused Assignment Tests

### DISCUSSION THREAD

**Discussion by chrissphinx** [community/community#67487] [GitHub Education]
### Hi everyone,

Hello GitHub Classroom team, it looks like the autograding editing modal dialog has been replaced with a fancy sliding window from the right side of the screen, however clicking on the **EDIT PENCIL** icons for previously made tests such as those on reused assignments will not bring up **anything**.

Going back to the sourced assignment (the one that was chosen to be reused) it appears some tests are viewable/editable but others are not. I'm pretty sure I used the same type of test for each one, but perhaps some test types have changed? Maybe with the introduction of the new output comparison options?

Please help, I'm not sure how this tests was entered and I cannot recreate it properly without being able to view what was entered.

### A suggestion...(optional)

uh definitely need some kind of bug report system for github classroom please, the discussion forms is not appropriate for these reports

**Comment by chrissphinx** [comment]
Actually bu appears worse than this as I cannot entered newly created tests, so it seems I'm unable to add tests to assignments properly now...

**Reply by chpurdy** [reply]
I posted about a similar issue on Wednesday last week (https://github.com/orgs/community/discussions/67052)

I was able to create a test using "echo" in the run command.  For example, if you want to read 15 from stdin, you would do the following run command (assuming Java).

echo 15 | java ProgramName

The same should work for other languages (I think).

I also agree that this forum is _not_ a good way to share bugs like this!
